@here/harp-mapview/lib/MapView.ts

```diff
@@ -1505,11 +1516,40 @@ export class MapView extends THREE.EventDispatcher {
         return this.m_focalLength;
     }
 
-    /**
+    /** @internal
      * Get distance from camera to the point of focus in world units.
+     *
+     * @note If camera does not point to any ground anymore the last focus point distance is
```

Comment:
I still don't like this b/c it introduces in-deterministic behavior but we will address it in the follow up ticket

Reply:
Ok. thanx!